components/toast/README.md

```diff
@@ -0,0 +1,168 @@
+you wish to change and include them after the default styles.
+
+**You can put any custom styles inline or use class names and add an external file.**
+
+### Classes you can override
+
+| Class Name | What it styles |
+|------------|----------------|
+|guic-toast-container|The containers the toasts are nested inside|
+|guic-toast| The toast component itself|
+|guic-toast-message|The message slot styles|
+|guic-toast-close-btn|The style while pressing the button|
+|guic-toast-top/bottom/right/left/center|The styles used on the containers for the positioning of the toasts|
+
+### Overriding the animations
+
+There are 2 default animations set:
+
+- **Slide Up/Down Animation**: By default, the toast uses slide-up and slide-down animations when shown. This behavior can be overridden by applying a custom class with a CSS specificity higher than `0,1,1` and defining your own animations. Alternatively, you can modify the existing animations named `guic-toast-slide-up` and `guic-toast-slide-down` in your CSS. For example:
+  ```css
+  gameface-toast.custom-toast-class {
+      animation-name: customSlideUp;
+  }
+  @keyframes customSlideUp {
+      from { transform: translateY(100%); opacity: 0; }
+      to { transform: translateY(0); opacity: 1; }
+  }
+- **Fade Out Animation**: The default fade-out animation can be overridden by using the `guic-toast-hide` class along with a custom animation named `guic-toast-fade-out`. Ensure that any custom animations maintain the same name (`guic-toast-fade-out`) as the component’s `hide` method is specifically designed to listen for this animation's end event. To effectively override the default, your custom CSS rules must have a specificity greater than `0,1,1`. Here’s how you could define a custom fade-out animation:
+  ```css
+  .custom-toast-class.guic-toast-hide {
+      animation-name: guic-toast-fade-out; /* Use the same name for compatibility */
+      animation-duration: 0.8s; /* Custom duration */
+  }
+  @keyframes guic-toast-fade-out {
+      from { opacity: 1; }
+      to { opacity: 0; }
+  }
+  ```
+
+Specifying the content
+=========================
+
+Use the `message` slot to specify the message of the toast.
+
+~~~~{.html}
+<gameface-toast gravity="top" position="left">
+    <div slot="message">Toast on top left</div>
+</gameface-toast>
+~~~~
+
+Use the `close-btn` slot to include a close button in your toast:
+
+~~~~{.html}
+<gameface-toast class="styled-toast" gravity="top" position="right" timeout="5000">
+  <div slot="message">I have a target element</div>
+  <div slot="close-btn">x</div>
+</gameface-toast>
+~~~~
+
+Keep in mind that you **must** provide some content inside the slot, otherwise the close functionality of the button won't work. That's because the slot expects some content inside and then attaches the closing logic. If you don't specify a `close-btn` slot the toast will require manual removal (if timeout attribute is missing).
```

Comment:
```suggestion
Keep in mind that you **must** provide some content inside the slot, otherwise the close functionality of the button won't work. That's because the slot relies on content being present to attach the closing logic. 
If you omit the `close-btn` slot, you'll need to manually handle the toast's removal (if the timeout attribute is not set).
```

Reply:
Fixed